src/pages/Search/SearchBooks.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import '../../styles/theme';
+import styled from 'styled-components';
+
+const filters = [
+  { filter: '전체', placeholder: '검색어를 입력하세요' },
+  { filter: '저자', placeholder: '저자를 입력하세요' },
+  { filter: '제목', placeholder: '제목을 입력하세요' },
+  { filter: '출판사', placeholder: '출판사를 입력하세요' },
+];
+
+export default function SearchBooks() {
+  const [filter, setFilter] = useState('전체');
+  const [visible, setVisible] = useState(false);
```

Comment:
```suggestion
  const [isFilterVisible, setIsFilterVisible] = useState(false);
```
- 어떤 대상이 보이고 보이지 않는지 명시해주는게 좋아보입니다
- boolean 타입의 변수는 is- prefix를 붙여주면 좀 더 변수 파악이 쉬울거 같아요

Reply:
수정 완료했습니다